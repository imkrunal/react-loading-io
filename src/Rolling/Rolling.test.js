import React from 'react'
import serializer from 'jest-emotion'
import renderer from 'react-test-renderer'
import { Rolling } from '..'

expect.addSnapshotSerializer(serializer)

it('rolling renders correctly', () => {
  const tree = renderer.create(<Rolling />).toJSON()
  expect(tree).toMatchSnapshot()
})
