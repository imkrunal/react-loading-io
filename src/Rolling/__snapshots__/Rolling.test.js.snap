// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rolling renders correctly 1`] = `
@keyframes animation-0 {
  0% {
    -webkit-transform: translate(-50%,-50%) rotate(0deg);
    -ms-transform: translate(-50%,-50%) rotate(0deg);
    transform: translate(-50%,-50%) rotate(0deg);
  }

  100% {
    -webkit-transform: translate(-50%,-50%) rotate(360deg);
    -ms-transform: translate(-50%,-50%) rotate(360deg);
    transform: translate(-50%,-50%) rotate(360deg);
  }
}

.emotion-1 {
  position: relative;
  width: 200px !important;
  height: 200px !important;
  -webkit-transform: translate( -200px,-200px ) scale(1) translate( 200px,200px );
  -ms-transform: translate( -200px,-200px ) scale(1) translate( 200px,200px );
  transform: translate( -200px,-200px ) scale(1) translate( 200px,200px );
}

.emotion-0 {
  position: absolute;
  width: 160px;
  height: 160px;
  border: 20px solid #f08d43;
  border-top-color: transparent;
  border-radius: 50%;
  -webkit-animation: animation-0 1s linear infinite;
  animation: animation-0 1s linear infinite;
  top: 100px;
  left: 100px;
  -webkit-transform: rotate(90deg);
  -ms-transform: rotate(90deg);
  transform: rotate(90deg);
}

<div>
  <div
    className="emotion-1"
  >
    <div
      className="emotion-0"
    />
  </div>
</div>
`;
